AmericanExpress/src/test/java/com/braintreepayments/api/AmericanExpressClientUnitTest.java

```diff
@@ -114,6 +122,27 @@ public void getRewardsBalance_callsListenerWithRewardsBalanceWithErrorCode_OnIns
         assertEquals("Insufficient points on card", rewardsBalance.getErrorMessage());
     }
 
+    @Test
+    public void getRewardsBalance_callsBackFailure_OnHttpError() {
```

Comment:
love that we're adding this test coverage. Question: do we have testing around analytic events being sent?